src/svalue.h

```diff
@@ -248,18 +248,26 @@ struct svalue_s
    *   Operator-closure index = CLOSURE_OPERATOR_OFFS + instruction index
    *   Efun-closure index     = CLOSURE_EFUN_OFFS + instruction index
    *   Simul_efun index       = CLOSURE_SIMUL_EFUN_OFFS + function index
-   * Yes, the operator range can be overlaid onto the efun range without
+   * Yes, the operator range could be overlaid onto the efun range without
    * collision, distinguishing them by the struct instr[].Default fields
    * (the 'closure' instruction does that to save space), but this way they
    * are easier to distinguish.
    * TODO: Note: Some code interprets these values as unsigned shorts.
    */
 
-#define CLOSURE_OPERATOR        (-0x1800)  /* == 0xe800 */
+#ifdef USE_PYTHON
+#  define CLOSURE_OPERATOR      (-0x2000)  /* == 0xe000 */
```

Comment:
This change must also be ported to /mudlib/sys/lpctypes.h,
for this we need a __PYTHON__ define in lex.c